src/shared/utils/exportData/prepareData.ts

```diff
@@ -18,9 +24,21 @@ export const prepareData = (
   const flattenAnswers = parsedAnswers
     .flatMap((item) => item.decryptedAnswers)
     ?.filter((item) => !item.activityItem?.config?.skippableItem);
+  const flattenEvents = parsedAnswers.flatMap((item) => {
+    const decryptedAnswersObject = getDecryptedAnswersObject(item.decryptedAnswers);
```

Comment:
It is not possible to do that in the parsedAnswers
    .flatMap topwards ?

Reply:
there are no way to do it properly in one place in flatMap